pkg/sink/configs.go

```diff
@@ -0,0 +1,6 @@
+package sink
+
+const (
+	Protocol = "unix"
+	Addr     = "/var/run/numaflow/function.sock"
+)
```

Comment:
I first refactored the configs to be pkg/configs
then when I was working on the numaflow project I found that I need to import one more pkg/configs
so I reverted the refactor and added a separate configs under sink

let me know what do you think